script/lib/app.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = exports = {};
+
+// let name = 'Kayla';
```

Comment:
Won't dock points now for this, but  make sure to take out commented code before submitting.